pkg/actors/internal/placement.go

```diff
@@ -304,15 +305,8 @@ func (p *ActorPlacement) closeStream() {
 		p.clientLock.Lock()
```

Comment:
Another (unrealted) thing is that we should switch from "uber/atomic" to the built-in atomic package in the stdlib